<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortestTripDialog</class>
 <widget class="QDialog" name="ShortestTripDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plan Shortest Trip from Dodger Stadium</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="selectAllButton">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deselectAllButton">
       <property name="text">
        <string>Deselect All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="planTripButton">
       <property name="text">
        <string>Plan Trip</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="routeLabel">
     <property name="text">
      <string>Route:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="routeList"/>
   </item>
   <item>
    <widget class="QLabel" name="distanceLabel">
     <property name="text">
      <string>Total Distance: 0 miles</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 